kfdef/source/v1.0.2/kfctl_aws.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   repos:
   - name: manifests
-    uri: https://github.com/kubeflow/manifests/archive/v1.0-branch.tar.gz
+    uri: https://github.com/kubeflow/manifests/archive/v1.0.2.tar.gz
```

Comment:
Hi @richardsliu, Can you help remove kfserving-istiogateway in AWS manifest as well? 

We are not clear about how to deal with this early this week, so we just add to 1.0.2 AWS.